File: tests_bench/recursive_fib.lua
local start = os.clock()
local function fib(n)
    if n < 2 then
        return n
    end
    return fib(n - 1) + fib(n - 2)
end


fib(30)


local elapsed = os.clock() - start
print(elapsed * 1000) 

File: examples/python/recursive_fib.py
import time

def fib(n: int) -> int:
    if n < 2:
        return n
    return fib(n - 1) + fib(n - 2)


start = time.perf_counter()
fib(30)

end = time.perf_counter()
print((end - start) * 1000)  
